pvlib/pvsystem.py

```diff
@@ -693,12 +695,12 @@ def sapm(module, poa_direct, poa_diffuse, temp_cell, airmass_absolute, aoi):
         (1 + module['Aimp']*(temp_cell - T0)) )
 
     dfout['v_oc'] = (( module['Voco'] +
-        module['#Series']*delta*np.log(Ee) + Bvoco*(temp_cell - T0) )
+        module['Cells in Series']*delta*np.log(Ee) + Bvoco*(temp_cell - T0) )
```

Comment:
I take it back. I don't think it's worth all of the changes to make them lowercase, which is why I didn't do it in the first place. I'll replace the spaces, though, since it's only a few variables.